### ISSUE TITLE
Any plans for further support?

### ISSUE THREAD

**Issue by gallyamb** [LinFor/jellyfin-plugin-kinopoisk#40]
First of all, thank you for all of your efforts on creation and maintaining such an amazing plugin! That's really what the Russian community is needed

Just want to clarify, do you have a time for further support of this repository? There are a couple of issues and pull requests without your replies. Maybe, we should find one more maintainer?

**Comment by LinFor** [comment]
Привет, вроде вся аудитория должна быть русскоязычной, так-то на русском.

Короткий ответ - нет, планов по сколько-нибудь стабильной поддержке в обозримом будущем нет.

Касательно мейнтейнеров - я совершенно не против, если найдутся желающие. Могу дать права на эту репу, могу жирно в ридми рекомендовать использовать другую репу. Но пока насколько я вижу в форках прям стабильно поддерживаемых вроде нет (могу ошибаться). Так что если у кого есть желание с этим возиться - пишите сюда.

**Comment by gallyamb** [comment]
А в чем была изначальная причина "заброса" репы? Банальное отсутствие времени, или Кинопоиск вставляет палки в колёса изменениями в API? Просто нехватка времени кажется менее сложной для преодоления проблемой, если вдруг найдется желающий продолжить дело. А вот с сопротивлением со стороны Кинопоиска работать уже значительно сложнее

**Comment by LinFor** [comment]
> А в чем была изначальная причина "заброса" репы? Банальное отсутствие времени, или Кинопоиск вставляет палки в колёса изменениями в API? Просто нехватка времени кажется менее сложной для преодоления проблемой, если вдруг найдется желающий продолжить дело. А вот с сопротивлением со стороны Кинопоиска работать уже значительно сложнее

Нет, никакого сопротивления кинопоиска не было, я думаю они вообще не в курсе об этом проекте. По крайней мере не контактировали со мной ни разу.
Неофициальное API, которое тут используется - не идеальное, да, в том смысле, что версионирование изменений обеспечивает примерно никак. С чем в общем-то и связана постоянная потеря работоспособности плагина. Но уж какое есть, сам-то кинопоиск вообще не предоставляет api для коммунити-проектов.

Плагин я этот делал для себя, пока это было интересно - рейтинг кинопоиска ещё что-то значил, каталог старались поддерживать актуальным. Сейчас кинопоиск это в первую очередь стриминг-сервис, коих стало много, их рейтинг перестал кореллировать с личными ощущениями, а сериалы сторонних стримингов на кинопоиске часто появляются с порядочной задержкой.

С учётом этих изменений, лично для меня плагин фактически потерял актуальность (хотя я по инерции продолжаю им пользоваться, закрывая глаза на ошибки получения метаданных для фильмов). Интересным сейчас кажутся другие источники данных - https://ru.kinorium.com, https://myshows.me, либо отдельные плагины для каждой из отечественных стримингов.

Но, повторюсь, вероятность, что я чем-то из этого займусь в обозримые года два - околонулевая. Есть работа, есть стройка, есть куча более актуальных для меня, но недоделанных проектов, в общем - без шансов :)

Желающие продолжить это дело - велкам в любом виде, код можете считать public domain (если кого-то беспокоят вопросы лицензии), как бы пафосно это не звучало, т.к. никакой ценности в нём вроде нет :)

**Comment by gallyamb** [comment]
Про кинориум, кстати, да, крутой сайт! Я им даже писал по поводу их желания сделать/дать API для реализации плагина. Но они чот не ответили

Понял, спасибо за подробности! Удачи в проектах и стройке :)

**Comment by gallyamb** [comment]
@LinFor что думаешь?

**Comment by LinFor** [comment]
@kobzantsev @gallyamb так я только за, кому дать права?

**Comment by walther555** [comment]
> @kobzantsev @gallyamb так я только за, кому дать права?

А вам не кажется, что пора в архив отправить репозиторий? Обновлений плагина нет больше полугода. Уже 12 версия Jellyfin вышла! Она не поддерживает плагин текущей версии.
